src/server.rs

```diff
@@ -18,7 +18,7 @@ use crate::cli;
 
 /// Axum server main entry point
 pub async fn main(args: cli::Args) {
-    setup_logging();
+    setup_logging(&args);
```

Comment:
`setup_logging` should only take what it needs -- verbosity level -- not the whole args.

As a general rule, a function must take only the arguments it needs.